core/src/test/scala/scalgorand/domain/asset/AssetsSpec.scala

```diff
@@ -0,0 +1,57 @@
+package scalgorand.domain.asset
+
+import zio.Scope
+import zio.json.DecoderOps
+import zio.test.{Spec, TestEnvironment, ZIOSpecDefault, assertTrue}
+
+object AssetsSpec extends ZIOSpecDefault {
+  override def spec: Spec[TestEnvironment with Scope, Any] =
+    test("Test for Assets Serialization") {
+      val assetsData =
+        """
+          |{
+          |  "assets": [
+          |    {
+          |      "created-at-round": 95070,
+          |      "deleted": false,
+          |      "index": 1,
+          |      "params": {
+          |        "clawback": "RZW5QPI4X7NZXUKFJZXOU3F5QQR7Q7MW5ZG3467XDWTMGO3YDUW3YCIJPQ",
+          |        "creator": "RZW5QPI4X7NZXUKFJZXOU3F5QQR7Q7MW5ZG3467XDWTMGO3YDUW3YCIJPQ",
+          |        "decimals": 0,
+          |        "default-frozen": false,
+          |        "freeze": "RZW5QPI4X7NZXUKFJZXOU3F5QQR7Q7MW5ZG3467XDWTMGO3YDUW3YCIJPQ",
+          |        "manager": "RZW5QPI4X7NZXUKFJZXOU3F5QQR7Q7MW5ZG3467XDWTMGO3YDUW3YCIJPQ",
+          |        "name": "asset1",
+          |        "name-b64": "YXNzZXQx",
+          |        "reserve": "RZW5QPI4X7NZXUKFJZXOU3F5QQR7Q7MW5ZG3467XDWTMGO3YDUW3YCIJPQ",
+          |        "total": 10000
+          |      }
+          |    },
+          |    {
+          |      "created-at-round": 120860,
+          |      "deleted": false,
+          |      "index": 15,
+          |      "params": {
+          |        "clawback": "4FK4BMFSAIA65UDBXVDZIRBKKY3D2PGQMSQ2NSKVMK4Z3ZOEMCHVA6KI6I",
+          |        "creator": "4FK4BMFSAIA65UDBXVDZIRBKKY3D2PGQMSQ2NSKVMK4Z3ZOEMCHVA6KI6I",
+          |        "decimals": 0,
+          |        "default-frozen": false,
+          |        "freeze": "4FK4BMFSAIA65UDBXVDZIRBKKY3D2PGQMSQ2NSKVMK4Z3ZOEMCHVA6KI6I",
+          |        "manager": "4FK4BMFSAIA65UDBXVDZIRBKKY3D2PGQMSQ2NSKVMK4Z3ZOEMCHVA6KI6I",
+          |        "reserve": "4FK4BMFSAIA65UDBXVDZIRBKKY3D2PGQMSQ2NSKVMK4Z3ZOEMCHVA6KI6I",
+          |        "total": 10000,
+          |        "unit-name": "bcoin",
+          |        "unit-name-b64": "YmNvaW4="
+          |      }
+          |    }
+          |  ],
+          |  "current-round": 20332345,
+          |  "next-token": "15"
+          |}
+          |""".stripMargin
+
```

Comment:
Same as above, We can inline any Json less than 10 to 15 lines

Reply:
Addressed in the latest commit. I have also updated Block , Status, and PendingTransactions where the Json data has more than 10 lines.